### ISSUE TITLE
AttributeError when exploring a sketch

### ISSUE THREAD

**Issue by hiddenillusion** [google/timesketch#232]
Looks like the changes [here](https://github.com/google/timesketch/commit/7fb1c9ae4daa4c52c0c4f27a6ef8755e24f5af31#diff-8c0f1f9e34c81d02779ee47294d6f7a3L200) cause an AttributeError to be raised. If one creates a sketch and clicks on the `Explore` tab when there aren't any views, the logic attempts to use `validate_filter` on `view` when it doesn't exist.

Traceback:

```
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2000, in __call__
return self.wsgi_app(environ, start_response)
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1991, in wsgi_app
response = self.make_response(self.handle_exception(e))
File "/usr/local/lib/python2.7/dist-packages/flask_restful/__init__.py", line 271, in error_router
return original_handler(e)
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1567, in handle_exception
reraise(exc_type, exc_value, tb)
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1988, in wsgi_app
response = self.full_dispatch_request()
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1641, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/usr/local/lib/python2.7/dist-packages/flask_restful/__init__.py", line 271, in error_router
return original_handler(e)
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1544, in handle_user_exception
reraise(exc_type, exc_value, tb)
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1639, in full_dispatch_request
rv = self.dispatch_request()
File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1625, in dispatch_request
return self.view_functions[rule.endpoint](**req.view_args)
File "/usr/local/lib/python2.7/dist-packages/flask_login.py", line 792, in decorated_view
return func(*args, **kwargs)
File "/opt/timesketch/timesketch/ui/views/sketch.py", line 199, in explore
query_filter = view.validate_filter(dict(indices=sketch_timelines))
AttributeError: 'NoneType' object has no attribute 'validate_filter'
```

**Comment by berggren** [comment]
Good catch! This is fixed in #233 and the CL is pending review.

**Comment by berggren** [comment]
Fixed
